main.css

```diff
@@ -636,11 +636,53 @@ p {
   text-align: center;
 }
 
+/* mobile popup */
+.popup-page {
+  position: fixed;
+  background-color: #6070ffd5;
+  backdrop-filter: blur(6px);
+
+  /* opacity: 0.98; */
```

Comment:
- [ ] Please, kindly remove this old piece of commented out code, to improve the cleanliness of the repo 💪